bin/crew

```diff
@@ -1087,9 +1087,9 @@ def prepare_package(destdir)
     end
 
     # compress manual files
-    compress_doc "#{CREW_DEST_PREFIX}/man"
+    compress_doc CREW_DEST_MAN_PREFIX.to_s
     compress_doc "#{CREW_DEST_PREFIX}/info"
-    compress_doc "#{CREW_DEST_PREFIX}/share/man"
+    compress_doc "#{CREW_DEST_PREFIX}/man"
```

Comment:
This is wrong, correct?

Reply:
I've just sorted the list...

Reply:
We definitely don't want to compress man pages in `/usr/local/man`.

Reply:
Too many packages store files there. See https://github.com/chromebrew/chromebrew/issues/8275

Reply:
But we don't want new packages to store manpages in this location, right?

Reply:
This is fixed.